pipelines/utilities/run_longhorn_e2e_test.sh

```diff
@@ -63,3 +63,16 @@ run_longhorn_e2e_test(){
   kubectl cp ${LONGHORN_TEST_POD_NAME}:/tmp/test-report/output.xml "output.xml" -c longhorn-test-report
   kubectl cp ${LONGHORN_TEST_POD_NAME}:/tmp/test-report/report.html "report.html" -c longhorn-test-report
 }
+
+run_longhorn_e2e_test_out_of_cluster(){
+  cd e2e
+  pip install -r requirements.txt
```

Comment:
If running test cases, out of cluster, we should still use container to run it independent of the host. You can assume docker here or any container manager CLI (nerdctl) you prefer.

Reply:
It already runs in a container. Reference https://github.com/longhorn/longhorn-tests/blob/master/pipelines/e2e/Jenkinsfile#L75 for more details.